packages/stash/src/actions/getConfig.ts

```diff
@@ -0,0 +1,14 @@
+import { Table } from "@latticexyz/config";
+import { Stash } from "../common";
+
+export type GetConfigArgs = {
+  stash: Stash;
+  table: { label: string; namespaceLabel?: string };
+};
+
+export type GetConfigResult<table extends Table = Table> = table;
+
+export function getConfig({ stash, table }: GetConfigArgs): GetConfigResult<Table> {
```

Comment:
is this still necessary if we're using full table objects now?

Reply:
The idea with this one is being able to get the config from a table in situations where it's dynamically synced from a store registration log. Will rename to `getTableConfig`